### ISSUE TITLE
`clock.getDelta()` sometimes returns `0` which leads to division by `0` issues

### ISSUE THREAD

**Issue by pmi-gbs** [NASA-AMMOS/3DTilesRendererJS#737]
**Describe the bug**

I've noticed that in the `EnvironmentControls`, which broke the camera transform (`NaN` and the likes) and other values of it.

in:

```js
update( deltaTime = Math.min( this.clock.getDelta(), 64 / 1000 ) )
```
It should probably defused right after this line (and other lines where a `deltaTime` is used) with something like:
```js
deltaTime = Math.max( deltaTime, 0.000001 );
```
**To Reproduce**

Not sure, to be honest, took some time to find out, that `getDelta()` is allowed to return `0`, it randomly occured, while moving around with the controls. It could be that `update()` was executed consecutively, but even then I don't think it should break the camera.

**Comment by gkjohnson** [comment]
Thanks - are you passing "deltaTime" in yourself? Or relying on the built in clock? Either way after testing by directly passing 0 in in a second `update` call it looks like this is causing an issue. I think an easy solution here is to just return early from the update function if is deltaTime <= 0. Would you like to make a PR?

**Comment by gkjohnson** [comment]
Fixed in #743. Without any more information about how this is happening in practice (update must be getting called back to back) checking for 0 seems like the most reasonable solution.

**Comment by pmi-gbs** [comment]
Yes it did happen with the default `clock` and my own clock as well. I wonder why though, as even consecutive calls of `clock.getDelta()` should intuitively return a value `> 0`, but I guess the precision of `performance.now()` is kinda bad (and because of all the security stuff got even worse up to `> 100 us`)

The merged fix is even better thanks. I checked the rest of the code-base and indeed these were the only concerning `clock.getDelta()` calls (as otherwise only `delta * ...` is done).
